src/answers-umd.js

```diff
@@ -259,50 +259,29 @@ class Answers {
       if (parsedConfig.templateBundle) {
         this.renderer.init(parsedConfig.templateBundle);
       }
-
-      this._handlePonyfillCssVariables(
-        parsedConfig.disableCssVariablesPonyfill,
-        this._invokeOnReady.bind(this));
-      return this;
+    } else {
+      // Templates are currently downloaded separately from the CORE and UI bundle.
+      // Future enhancement is to ship the components with templates in a separate bundle.
+      this.templates = new DefaultTemplatesLoader(templates => {
+        this.renderer.init(templates);
+      });
     }
 
-    // Templates are currently downloaded separately from the CORE and UI bundle.
-    // Future enhancement is to ship the components with templates in a separate bundle.
-    this.templates = new DefaultTemplatesLoader(templates => {
-      this.renderer.init(templates);
-      this._handlePonyfillCssVariables(
-        parsedConfig.disableCssVariablesPonyfill,
-        this._invokeOnReady.bind(this));
+    const onReadyPromise = new Promise((resolve, reject) => {
+      this._handlePonyfillCssVariables(parsedConfig.disableCssVariablesPonyfill)
+        .then(this._masterSwitchApi.isDisabled())
+        .then((isDisabled) => {
```

Comment:
nit but I think our style is to normally omit the parenthesis around a javascript arrow function'
s param if there's only one param

Reply:
Will do!